auth-web/src/models/business.ts

```diff
@@ -122,6 +122,9 @@ export interface BusinessRequest {
         business?: {
             legalType: CorpTypes
         },
+        amalgamation?: {
+          nameRequest: NameRequest
```

Comment:
Please add `amalgamationType` here as well. Type is: https://github.com/bcgov/bcrs-shared-components/blob/main/src/enums/amalgamation-types.ts

Reply:
LGTM